client/gutenberg/extensions/slideshow/slideshow.js

```diff
@@ -151,21 +152,31 @@ class Slideshow extends Component {
 						className="wp-block-jetpack-slideshow_pagination swiper-pagination swiper-pagination-white"
 						ref={ this.paginationRef }
 					/>
-					<button
+					<a
 						className="wp-block-jetpack-slideshow_button-prev swiper-button-prev swiper-button-white"
 						ref={ this.btnPrevRef }
-					/>
-					<button
+						role="button"
+					>
+						{ __( 'Previous Slide' ) }
```

Comment:
Won't this `__()` lead to issues with block invalidations when site language changes?

Reply:
Yes. How have we been protecting against this?

Reply:
See pafL3P-kL-p2

Reply:
I outlined 2 alternatives here: pafL3P-kL-p2 #comment-1464

- Remove and PHP render the bits that are subject to change
- Push those bits into attributes to protect against invalidation.

Neither of them are very appealing, here or in many other cases. The latter essentially mimics this (we're already serializing localized strings that won't update with site language) but adds the safety that changes in translation or the site option will not invalidate existing blocks.

Reply:
and in tiled gallery we removed the label for now: https://github.com/Automattic/wp-calypso/pull/31119/files#diff-901bdd0db2c9f91df2138d1996ea0390L41

Reply:
Removed all of the localized button text in https://github.com/Automattic/wp-calypso/pull/31175/commits/b7a222c34df29e69943a78b9d9e413ff283d6198